modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/fragment/renderer/FDSViewFragmentRenderer.java

```diff
@@ -464,7 +464,7 @@ private JSONArray _getFiltersJSONArray(
 					);
 				}
 
-				String listTypeDefinitionId = MapUtil.getString(
+				Long listTypeDefinitionId = MapUtil.getLong(
```

Comment:
I'd swear this was a String ERC when I added this code... :thinking:

I remember it because I realized there was a mismatch between the naming and the content but didn't pay too much attention to it, TBH.

Have we changed it in another PR? :thinking:

Reply:
Looks like it was ID [from the start](https://github.com/liferay-frontend/liferay-portal/pull/3323/files). I'm not sure if there was some off-the-code intent to use ERC.